src/main/java/com/google/api/gax/grpc/BundlingSettings.java

```diff
@@ -155,6 +161,12 @@ public Builder setRequestByteThreshold(Integer requestByteThreshold) {
      */
     public abstract Builder setIsEnabled(Boolean enabled);
 
+    public abstract Builder setIsFlowControlEnabled(Boolean enabled);
+
+    public abstract Builder setFailOnFlowControlLimits(Boolean failOnFlowControlLimits);
```

Comment:
Does setting `setFailOnFlowControlLimits` to true differ meaningfully from `setIsFlowControlEnabled`? `getFailOnFlowControlLimits` doesn't appear to be used. I might recommend only using the "enabled" concept.